### PR TITLE
Delighted new action sending to people

### PR DESCRIPTION
**Pull request by jcortes**
This PR was uploaded by the user @nauxliu and has the fix reported at #2308 by QA

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/delighted/actions/sending-to-people/sending-to-people.mjs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import delightedApp from "../../delighted.app.mjs";
+
+export default {
+  key: "delighted-sending-to-people",
+  name: "Sending to people",
```
Comment (michelle0927) [inline components/delighted/actions/sending-to-people/sending-to-people.mjs:5]:
```suggestion
  name: "Sending to People",
```

File: components/delighted/actions/sending-to-people/sending-to-people.mjs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  description: "Create or update a person and send a survey email. [See the docs here](https://app.delighted.com/docs/api/sending-to-people)",
+  version: "0.0.7",
```
Comment (michelle0927) [inline components/delighted/actions/sending-to-people/sending-to-people.mjs:7]: Should be version 0.0.1
```suggestion
  version: "0.0.1",
```

File: components/delighted/delighted.app.mjs [hunk 3]
```diff
@@ -1,11 +1,17 @@
+import delightedLib from "delighted";
+
 export default {
   type: "app",
   app: "delighted",
+  label: "Delighted",
+  description: "The Delighted app that will be used to send the survey.",
```
Comment (michelle0927) [inline components/delighted/delighted.app.mjs:7]: These aren't needed.
```suggestion
```

**Review by michelle0927 — CHANGES_REQUESTED** [review]

**Review by michelle0927 — APPROVED** [review]

**Comment by dannyroosevelt** [comment]
This is ready for release!

**Pull request merged**